serverless/Cargo.toml

```diff
@@ -9,7 +9,7 @@ env_logger = "0.10.0"
 datadog-trace-mini-agent = { path = "../trace-mini-agent" }
 datadog-trace-protobuf = { path = "../trace-protobuf" }
 datadog-trace-utils = { path = "../trace-utils" }
-dogstatsd = { path = "../dogstatsd" }
+dogstatsd = { version = "0.1.0", default-features = false, git = "https://github.com/DataDog/serverless-components/", rev = "4dfe72ab1850680f41dd79d30a937eb68e7ba6da" }
```

Comment:
do you need the rev here? just curious, not sure how it works when you have both version and a git rev